### DISCUSSION TITLE
Using Static Search Plugin for static website

### DISCUSSION THREAD

**Discussion by TakumiFujiwara26** [GetPublii/Publii#1534] [Help] [answered]
I have a question about the Publii search plugin called "Static Search". I have a static website that is stored in Google Cloud Console and the idea of the website is more like a book where it consist of multiple folders for each chapter and each chapter folder holds the index.html file. I'm using the Publii only for the homepage, contact us, and FAQ pages and I want to know if using this Static Search plugin will crawl all the websites folder and files. The chapter pages are generated using a script and it uses the same styling as the pages from the actual Publii site, so they're sharing the same DNA. Just want to make sure before buying the plugin if it should work on my static website.

**Comment by bobmitro** [comment, marked as answer]
Unfortunately, it won't work with your files. The search index is generated based on the content of the Publii website.
